store/sqlstore/upgrade.go

```diff
@@ -1683,3 +1685,55 @@ func hasMissingMigrationsVersion600(sqlStore *SqlStore) bool {
 
 	return false
 }
+
+func upgradeDatabaseToVersion610(sqlStore *SqlStore) {
+	if hasMissingMigrationsVersion610(sqlStore) {
+		sqlStore.AlterColumnTypeIfExists("Sessions", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("ChannelMembers", "Roles", "text", "varchar(256)")
+		sqlStore.AlterColumnTypeIfExists("TeamMembers", "Roles", "text", "varchar(256)")
+	}
+
+	// if shouldPerformUpgrade(sqlStore, Version5380, Version600) {
```

Comment:
Just beware the versions here should be updated.

Reply:
@streamer45 @jwilander Would you be willing to make the update today as Ibrahim is off this week?

Reply:
```suggestion
	// if shouldPerformUpgrade(sqlStore, Version600, Version610) {
```